rust/crd/src/lib.rs

```diff
@@ -19,7 +20,7 @@ use stackable_operator::{
     config::{fragment, fragment::Fragment, fragment::ValidationError, merge::Merge},
     k8s_openapi::apimachinery::pkg::api::resource::Quantity,
     kube::{runtime::reflector::ObjectRef, CustomResource},
-    product_config_utils::{ConfigError, Configuration},
+    product_config_utils::Configuration,
```

Comment:
nit: Use `product_config_utils::{self, Configuration},` instead which allows us to remove the above import in line 8.

Reply:
good find! https://github.com/stackabletech/hive-operator/pull/462/commits/5dac97f3b7cb4450f48421a57c886812cd409a8f